afs/afs-core/src/main/java/com/powsybl/afs/TaskMonitor.java

```diff
@@ -35,16 +51,31 @@
         @JsonProperty("projectId")
         private final String projectId;
 
+        @JsonIgnore
+        private Future future;
+
+        @JsonProperty("cancelable")
+        private boolean cancelable;
+
+        public Task(String name,
+                    String message,
+                    long revision,
+                    String projectId) {
+            this(name, message, revision, projectId, false);
+        }
+
         @JsonCreator
         public Task(@JsonProperty("name") String name,
```

Comment:
Should we categorize this as breaking change, since old JSON messages will not be readable anymore ?

Reply:
yeah ok, but maybe we should configure jackson to allow for unknown properties DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES <- false ?
I don't know how to do this in non spring apps.. :/

Reply:
I think for this use case it's enough to mention the change in the release note, just in case. I don't expect this feature to be widely used.

Otherwise, I guess the configuration would happen in `JacksonDecoder` class (where we use the default powsybl `ObjectMapper` at the moment).

Reply:
@geofjamg would it be ok to add `.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)` to the default objectMapper ?

Reply:
as it allows to extend dtos without breaking the app

Reply:
> @geofjamg would it be ok to add `.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)` to the default objectMapper ?

I would say no because of unexpected effects on other codes. Most of the time you want to be warmed in that case because it is an input error.

Reply:
ok, it wasn't added anyway.